datafusion/core/tests/sql/subqueries.rs

```diff
@@ -275,7 +275,7 @@ async fn tpch_q17_correlated() -> Result<()> {
         "+--------------------+",
         "| avg_yearly         |",
         "+--------------------+",
-        "| 1901.3714285714286 |",
+        "| 190.13714285714286 |",
```

Comment:
I've not independently verified this result, but I believe @andygrove added CI to check the TPCH queries are returning the correct results, so I think this was just out of date